lib/webpack-cli.js

```diff
@@ -111,6 +111,13 @@ class webpackCLI extends GroupHelper {
         const HelpGroup = require('./groups/help');
         return new HelpGroup().outputHelp();
     }
+    async runVersion() {
```

Comment:
Why `async` if it doesn't have `await`?

Reply:
Was following the async/await convention for each method, will remove